src/components/SingleArt.jsx

```diff
@@ -0,0 +1,225 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getSingleArt } from '../actions/artsActions';
+
+/**
+ *  @returns {JSX} jsx
+ */
+export class SingleArt extends Component {
+  /**
+   * @param {function} render
+   *  @returns {JSX} jsx
+   */
+  componentDidMount() {
+    const { slug } = this.props.match.params;
+    this.props.getSingleArt(slug);
+  }
+
+  /**
+   * @param {function} render
+   *  @returns {JSX} jsx
+   */
+  render() {
+    const { isLoading, details } = this.props.art;
+    return (
```

Comment:
Please go ahead and break this down into multiple smaller components. This components is too long and hard to follow.